### ISSUE TITLE
set_rate_BC ambiguous when imposed on a set of pores

### ISSUE THREAD

**Issue by ma-sadeghi** [PMEAL/OpenPNM#1340]
`alg.set_rate_BC(pores=some_pores, value=1.5)` might imply that the rate value `1.5` is to be applied to all `some_pores` combined, but rather it is basically

````
[alg.set_rate_BC(pores=p, value=1.5) for p in some_pores]

**Comment by ma-sadeghi** [comment]
@jgostick What do you think?

**Comment by ma-sadeghi** [comment]
On a relevant note, it might be worth it to re-introduce the concept of super pore, i.e. a virtual pore with no resistance, only used to distribute mass/energy when setting a rate BC on a bunch of pores, but when the individual rates are not known, which is by the way the more practical case.

**Comment by jgostick** [comment]
You're setting the rate on each of the pores individually.  The 1.5 is basically broadcasting to all the pores in the numpy sense.  BUT, I agree that setting a rate on a group of pores is a missing feature.  I feel like we discussed this and you were against it for mathematical reasons.  If we were to readd the super pore option, how would the api work?  We used to have a mode='group' or 'single'

BTW, another way to get a 'group' rate BC is to add your own single boundary pore, attached to each pore on a given face, then set rate leaving that pore.  The problem in both cases is specifying the conductance of the 'new' super throats.
